test/azure-pipelines.yml

```diff
@@ -75,11 +77,16 @@ jobs:
       curl -L https://raw.githubusercontent.com/InsightSoftwareConsortium/ITK/dashboard/itk_common.cmake -O
     displayName: 'Fetch CTest driver script'
 
+  - bash: |
```

Comment:
What is this for? In order to turn module template into a module, cookiecutter needs to be applied once. I don't think we need to apply it again here. Same for instances below.

Reply:
@dzenanz I was following the example given in `ITKModuleTemplate`

These lines come from synchronizing with:
https://github.com/InsightSoftwareConsortium/ITKModuleTemplate/blob/master/azure-pipelines.yml#L80

Reply:
If you try to synchronize with it directly, some weirdness sneaks in, e.g. `ITKModuleTemplate` is used as a name instead of `ITKTwoProjectionRegistration`.

You should invoke cookie-cutter and enter this module's information (name `ITKTwoProjectionRegistration` etc). The result should then be used for syncing.